static/js/public/accordion.test.js

```diff
@@ -0,0 +1,142 @@
+import initAccordion, { initAccordionButtons } from "./accordion";
+
+// jsdom (in version that comes with jest) doesn't support .closest
+// so we need to polyfill that
+// shouldn't be needed when jsdom used by jest is updated to v11.12.0
+window.Element.prototype.closest = function(selector) {
```

Comment:
https://jestjs.io/docs/en/configuration#setupfiles-array can we do this instead? We could just include out polyfill file and everything should be auto-filled?